doc/releases/changelog-dev.md

```diff
@@ -114,6 +114,16 @@
   [(#3706)](https://github.com/PennyLaneAI/pennylane/pull/3706)
   [(#3730)](https://github.com/PennyLaneAI/pennylane/pull/3730)
 
+* The `coefficients` function and the `visualize` submodule of the `qml.fourier` module
+  now allow assigning different degrees for different parameters of the input function.
+  [#3005](https://github.com/PennyLaneAI/pennylane/pull/3005)
```

Comment:
should probably move this PR link to the end of all change text

Reply:
hmm, I guess most change descriptions I see are either very short or very long so I never really considered what you've got here. I'm convinced! Let's leave it 👍